foundation/foundation-mda/src/main/resources/templates/deployment/spark-operator/spark-application-base-values.yaml.vm

```diff
@@ -56,9 +58,11 @@ sparkApp:
         #if (${enablePySparkSupport})
         - org.apache.sedona:sedona-python-adapter-3.0_2.12:${versionSedona}
         - org.datasyslab:geotools-wrapper:${versionGeotools}
-        - org.postgresql:postgresql:${versionPostgresql}
         #end
         #end
+        #if (${enablePySparkSupport})
+        - org.postgresql:postgresql:${versionPostgresql}
+        #end
```

Comment:
I: Provide the postgres jdbc driver by default to match default connection properties, even if Sedona isn't present. Regardless of this change, users will have to alter this driver dependency if they choose not to use postgres.

Reply:
Shouldn't we make inclusion conditional on whether Postgres is the persistence of choice?

Reply:
Yes. That said, since the switch to defining the jdbc connection details in a krausening file, I wonder if it would be best not to pre-emptively import drivers or have default rdbms configs that assume postgresql as the default. 

We do expect the users to be able to use the aissemble framework to implement their relational db of choice - dependency management included, correct? I'm trying to make the call on what is too little or too much scaffolding as providing the correct drivers dynamically for every possible rdbms option seems like its own effort. What do you think @peter-mcclonski ?

Reply:
Right now, the only RDBMS options present in aissemble are Postgres and, technically, MySQL-- though MySQL is used for only a single purpose (backing the Hive Metastore).  I would say that it's reasonable to support Postgres as the default implementation of the RDBMS option, but that it should be done in a way such that it's possible for a project to change from that default.

Reply:
It still strikes me as bizarre that the condition for Postgres inclusion is based on whether we're running a pyspark pipeline, rather than whether we're running a pipeline with RDBMS storage

Reply:
Agreed - I added the RDBMS flag to the generator and changed it in the template.